protocols/noise/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "libp2p-noise"
+description = "Cryptographic handshake protocol using the noise framework."
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+edition = "2018"
+
+[dependencies]
+bs58 = "0.2"
+bytes = "0.4"
+byteorder = "1"
+curve25519-dalek = "1"
+futures = "0.1"
+lazy_static = "1.2"
+libp2p-core = { version = "0.2.0", path = "../../core" }
+log = "0.4"
+rand = "0.6"
+tokio-io = "0.1"
+
+[dependencies.snow]
+git = "https://github.com/mcginty/snow.git"
```

Comment:
Is the crates.io version lacking features?

Reply:
There was a slight inconvenience related to their `Random` trait which was removed on `master`. I could switch to the crates.io version with minor effort or do it before we publish next time.

Reply:
Version `0.5.0-alpha1` was just released to crates.io. The PR has been updated to use this version.